AArch64 Windows uses a five-argument __va_start, just like ARM.

https://bugs.llvm.org/show_bug.cgi?id=39090

diff --git a/clang/lib/Sema/SemaChecking.cpp b/clang/lib/Sema/SemaChecking.cpp
index 22483f8..53d9cb2 100644
--- a/clang/lib/Sema/SemaChecking.cpp
+++ b/clang/lib/Sema/SemaChecking.cpp
@@ -917,6 +917,7 @@ Sema::CheckBuiltinFunctionCall(FunctionDecl *FDecl, unsigned BuiltinID,
     switch (Context.getTargetInfo().getTriple().getArch()) {
     case llvm::Triple::arm:
     case llvm::Triple::thumb:
+    case llvm::Triple::aarch64:
       if (SemaBuiltinVAStartARMMicrosoft(TheCall))
         return ExprError();
       break;
